libs/core/src/scss/base/_html-table.scss

```
@@ -7,7 +7,8 @@ $row-height: 44px;
 // Color variables
 $default-divider-color: utils.get-color('medium');
 $border-bottom: 1px solid var(--kirby-divider-color, #{$default-divider-color});
-$highlight-color: utils.get-color('light');
+$hover-color: utils.get-color('light');
+$active-color: utils.get-color('semi-light');
```

```suggestion
$hover-color: utils.get-color('light');
$active-color: utils.get-color('semi-light');
```

Should be removed if unused